tests/emulator/src/vbridge_impl.cc

```diff
@@ -206,6 +227,11 @@ void VBridgeImpl::run() {
         throw TimeoutException();
       }
     }
+
+    if (exit_after_all_issued) {
+      LOG(INFO) << fmt::format("[{}] all insn in to_rtl_queue are issued, exiting", get_t());
+      exit(0);
```

Comment:
Avoid `exit`, throw exception, or return to exit loop, in order to ensure graceful cleanup

Reply:
Ack